boilerplate/app/components/Button.tsx

```diff
@@ -105,7 +117,7 @@ export function Button(props: ButtonProps) {
   }
 
   return (
-    <Pressable style={$viewStyle} accessibilityRole="button" {...rest}>
+    <Pressable style={$viewStyle} accessibilityRole="button" {...rest} disabled={disabled}>
       {(state) => (
         <>
           {!!LeftAccessory && <LeftAccessory style={$leftAccessoryStyle} pressableState={state} />}
```

Comment:
Similar to the `disabledTextStyle` comment, for consistency, we should pass the `disabled` state into these accessories so the user has a choice to style them via these props rather than higher ordered state. 

`{!!LeftAccessory && <LeftAccessory style={$leftAccessoryStyle} pressableState={state} disabled={disabled} />}`

Example:
```tsx
<Button
  disabled={someComplex.any(logic => !!logic)}
  LeftAccessory={(props) => <Icon style={[props.disabled && $disabledIconStyle]} />}
```

Reply:
Ah yes, love it. Updated in https://github.com/infinitered/ignite/pull/2593/commits/d329a0cf8928498278e4de3b2703048623ce8090

Reply:
Whoops, forgot to update the interface. I'll do that along with the docs/examples updates in a push later on.